app/views/requirements/_requirement_step_fields.html.erb

```diff
@@ -2,6 +2,6 @@
 
 <div class="requirement_steps nested-fields">
```

Comment:
Yea, we've got a whole bunch of code from several versions ago; I say we clear it out (that's what git is for :-D).